tests/operator/operator.go

```diff
@@ -3423,3 +3407,18 @@ func allPodsAreTerminated(kv *v1.KubeVirt) {
 		}
 	}).WithTimeout(120 * time.Second).WithPolling(1 * time.Second).Should(Succeed())
 }
+
+func waitForUpdateCondition(kv *v1.KubeVirt) {
+	Eventually(func(g Gomega) *v1.KubeVirt {
+		foundKV, err := kubevirt.Client().KubeVirt(kv.Namespace).Get(context.Background(), kv.Name, metav1.GetOptions{})
+		g.Expect(err).ToNot(HaveOccurred())
+
+		return foundKV
+	}).WithTimeout(120 * time.Second).WithPolling(1 * time.Second).Should(
+		SatisfyAll(
+			matcher.HaveConditionTrue(v1.KubeVirtConditionAvailable),
+			matcher.HaveConditionTrue(v1.KubeVirtConditionProgressing),
+			matcher.HaveConditionTrue(v1.KubeVirtConditionDegraded),
+		),
+	)
+}
```

Comment:
does it make sense to make this exposed so that other tests could use it as well?

Reply:
Then I'm not sure it's the right place. that can be done in follow up PR.